src/Email/EmailRepository.php

```diff
@@ -211,7 +211,7 @@ public function add(Email $email)
             'datetime',
             'string',
             'string',
-            'integer',
+            'string',
```

Comment:
This was a nasty bug: all thread IDs were turned into 0.

I think because we had so many problems deploying with platform.sh we may have been running a very old version of the code for a long time (before we migrated away from it).

Reply:
Maybe this is something you can check with e.g. https://github.com/webmozart/assert and let psalm detect these errors with a CI build